ui/proxy.go

```diff
@@ -37,24 +36,24 @@ func (h *handler) imageProxy(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	clt := client.New(string(decodedURL))
+	clt := client.New(string(decodedURL)).WithStreaming(true)
 	resp, err := clt.Get()
 	if err != nil {
 		html.ServerError(w, r, err)
 		return
 	}
+	defer resp.Body.Close()
```

Comment:
The response body must be closed even when there is an error. In some cases, that might introduce a leak. For example, when there is a redirection failure, both err and resp will be non-nil.

See http://devs.cloudimmunity.com/gotchas-and-common-mistakes-in-go-golang/index.html#close_http_resp_body and previous fix: https://github.com/miniflux/miniflux/commit/7914261437305c3eb2ad2f1c750be91d95af718c#diff-6e3628bed2dbcc5fb97172548a186d5d